specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/datafactory.json

```diff
@@ -6467,6 +6707,167 @@
       "required": [
         "properties"
       ]
+    },
+    "PrivateEndpointConnectionListResponse": {
+      "description": "A list of linked service resources.",
+      "type": "object",
+      "required": [
+        "value"
+      ],
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "List of Private Endpoint Connections.",
+          "items": {
+            "$ref": "#/definitions/PrivateEndpointConnectionResource"
+          }
+        },
+        "nextLink": {
+          "description": "The link to the next page of results, if any remaining results exist.",
+          "type": "string"
+        }
+      }
+    },
+    "PrivateEndpointConnectionResource": {
```

Comment:
>PrivateEndpointConnectionResource": [](start = 5, length = 35)

I see you are defining resource definition in this file. Recommend referring to common definitions "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/..." this forces you to utilize the newer contract. Please create followup task to fix this in the next version.

Reply:
These apis only specific to DataFactory, so we do not need to refer to common definitions.